indico/modules/rb_new/client/js/components/modals/BookRoomModal.jsx

```diff
@@ -145,6 +145,7 @@ export default class BookRoomModal extends React.Component {
             const {booking} = this.state;
             const bookingLink = (
                 // TODO: add link to view booking details
+                // eslint-disable-next-line no-alert
```

Comment:
i intentionally didn't disable the rule since we want to remove the alert eventually